Map/RoomUI.gd

```diff
@@ -0,0 +1,10 @@
+extends Control
+
+var room: RoomBase
+var label: Label
```

Comment:
Alternatively can do `@export var label: Label` and then assign in the inspector. Then you could remove the `get_child(0).get_child(0)` call in `_ready`. This may be a bit better because we won't have to change the code if the hierarchy changes